azext_iot/_params.py

```diff
@@ -909,6 +909,12 @@ def load_arguments(self, _):
     with self.argument_context("iot central app") as context:
         context.argument("properties", arg_type=event_msg_prop_type)
 
+    with self.argument_context("iot central app show-provisioning-info") as context:
+        context.argument("app_id", options_list=["--app-id"], help="Target App.")
+        context.argument(
+            "device_id", options_list=["--device-id", "-d"], help="Target Device."
```

Comment:
`device_id` is already defined in parent command scope `iot`. Same with `app_id` in `iot central`. We don't want to redefine arguments if we don't have to.